server/app/services/languageProcessor/TaskMap.java

```diff
@@ -88,6 +87,24 @@ public JsonNode getTicketAssignee(String issueKey, JsonNode responseBody) {
   }
 
   /**
+   * This method requests status of issue and returns it to the calling method
+   *
+   * @param issueKey
+   * @return
+   */
+  public JsonNode IssueStatus(String issueKey, JsonNode responseBody) {
+    if (Extractor.extractString(responseBody, "status").equals("[\"Issue Does Not Exist\"]")) {
+      return parseToJson("fail", configuration.getString("error-message.issue-not-found"));
+    } else {
+      System.out.println("I am here...checkpoint 2");
+      String answer = "The status of" + issueKey + "is" + Extractor.extractString(responseBody, "status");
+      System.out.println("I am here...checkpoint 3");
```

Comment:
I think those System.out.println (e.g. `System.out.println("I am here...checkpoint 3");`) are for devlopment debug only? Please remove them if they are not related to the functionality or error output.